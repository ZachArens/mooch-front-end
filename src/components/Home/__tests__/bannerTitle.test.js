import React from 'react'
import renderer from 'react-test-renderer'
import BannerTitle from '../bannerTitle';

test('<BannerTitle />', () => {
    const tree = renderer.create(<BannerTitle />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});