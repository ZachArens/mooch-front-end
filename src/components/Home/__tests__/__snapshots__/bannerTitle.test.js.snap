// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BannerTitle /> 1`] = `
<div
  className="title-banner"
>
  <div
    className="input-group"
  >
    <h1>
      Mooch
    </h1>
    <input
      placeholder="Search"
      type="text"
    />
  </div>
  <div
    hidden={true}
  >
    Image by 
    <a
      href="https://pixabay.com/users/seaq68-4191072/?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=5621150"
    >
      Sven Lachmann
    </a>
     from 
    <a
      href="https://pixabay.com/?utm_source=link-attribution&utm_medium=referral&utm_campaign=image&utm_content=5621150"
    >
      Pixabay
    </a>
  </div>
</div>
`;
